src/components/TopicList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import TopicCard from "./TopicCard";
+import { getTopics } from "../utils/axios";
+
+const TopicList = () => {
+    const [topics, setTopics] = useState([])
+
+    useEffect(() => {
+        getTopics
+        .then((response) => {
+            return response.data
```

Comment:
This .then (and other similar) could be made part of your API call function. This would extract away the returning of the response data from your components and make things a bit cleaner.